src/main.rs

```diff
@@ -1,18 +1,40 @@
 extern crate ansi_term;
+extern crate atty;
 extern crate chrono;
 extern crate clap;
 extern crate strsim;
 extern crate todo_txt;
 
+use ansi_term::ANSIString;
 use ansi_term::Colour::{Blue, Green, Red};
 use chrono::{Datelike, Duration};
+use std::env;
 use std::fs::File;
 use std::io::{BufRead, BufReader};
 use std::str::FromStr;
 use strsim::levenshtein;
 use todo_txt::Task;
 use todo_txt::Date as TaskDate;
 
+
+fn is_a_tty() -> bool {
+    atty::is(atty::Stream::Stdout)
+}
+fn is_term_dumb() -> bool {
+    env::var("TERM").ok() == Some(String::from("dumb"))
+}
+
+macro_rules! color {
```

Comment:
Couldn't this macro be written as a function?

Reply:
You are correct. I had taken this macro from `clap` but it works well with a function.